pkg/cmd/repo/create/create.go

```diff
@@ -625,15 +626,14 @@ func interactiveRepoCreate(isDescEmpty bool, isVisibilityPassed bool, repoName s
 
 func confirmSubmission(repoName string, repoOwner string, inLocalRepo bool) (bool, error) {
 	qs := []*survey.Question{}
-
-	promptString := ""
+	var promptString string
+	targetRepo := normalizeRepoName(repoName)
+	if repoOwner != "" {
+		targetRepo = fmt.Sprintf("%s/%s", repoOwner, targetRepo)
+	}
```

Comment:
I moved the `targetRepo` assignment outside of the `if inLocalRepo` statement. Is this what you meant above about `HOST/REPO`?